etc/module.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="Ziffity_Task" setup_version="1.0.0">
```

Comment:
@NandhiniKarthi setup_version is deprecated in favor of declarative schema. why are you using this?